Maple2.Server.Game/Manager/HousingManager.cs

```diff
@@ -1,25 +1,35 @@
-﻿using System.Numerics;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
 using Maple2.Database.Extensions;
 using Maple2.Database.Storage;
 using Maple2.Model.Common;
 using Maple2.Model.Enum;
 using Maple2.Model.Error;
 using Maple2.Model.Game;
 using Maple2.Model.Metadata;
+using Maple2.PacketLib.Tools;
+using Maple2.Server.Core.Packets;
+using Maple2.Server.Game.Manager.Items;
+using Maple2.Server.Game.Model;
 using Maple2.Server.Game.Packets;
 using Maple2.Server.Game.Session;
 using Serilog;
+using Serilog.Core;
 
 namespace Maple2.Server.Game.Manager;
 
 public class HousingManager {
     private readonly GameSession session;
+    private readonly TableMetadataStorage tableMetadata;
     private Home Home => session.Player.Value.Home;
 
     private readonly ILogger logger = Log.Logger.ForContext<HousingManager>();
 
-    public HousingManager(GameSession session) {
+    public ItemBlueprint? StagedItemBlueprint = null;
```

Comment:
**suggestion:** Add documentation for StagedItemBlueprint property

The purpose and usage of the StagedItemBlueprint property are not immediately clear. Consider adding XML documentation to explain when and how this property is used.